pmd-core/src/main/java/net/sourceforge/pmd/renderers/XMLRenderer.java

```diff
@@ -152,6 +152,17 @@ public void end() throws IOException {
                 writer.write(buf.toString());
             }
         }
+        
+        // config errors
+        for (final Report.ConfigurationError ce : configErrors) {
+            buf.setLength(0);
+            buf.append("<configerror ").append("rule=\"");
```

Comment:
I did include a notice in the release changes, but feel free to edit it if you don't consider it explicit enough: https://github.com/pmd/pmd/pull/519/files#diff-b7709f07439de5204938d714fe93ed84

I'll get to the XML Schema on a separate PR / task